databroker/src/main.rs

```diff
@@ -457,7 +480,45 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
         apis.push(grpc::server::Api::SdvDatabrokerV1);
     }
 
-    grpc::server::serve(
+    let unix_socket = args.get_one::<String>("unix-socket").cloned();
+    if let Some(path) = unix_socket {
+        unlink_unix_domain_socket(&path)?;
```

Comment:
why do we need to unlink first?

Reply:
I think it is a quite common approach. Normally on a clean shutdown we will do the unlinking at exit, but for crashes and forced shutdown (`kill -9`) we will not unlink it, and then best to try to unlink it before creating it.

Reply:
understood

Reply:
Also, it will fail to bind the path if there is already a file in place.

But as you rightfully point out, a comment explaining the reason _why_ this is done would be appropriate :smile: